MunchkinLevelCounter/UI/Players Editor/PlayersEditorViewController.swift

```diff
@@ -190,18 +118,144 @@ class PlayersEditorViewController: BaseViewController, UITableViewDelegate, UITa
         warningAlert.addAction(cancelAction)
         present(warningAlert, animated: true, completion: nil)
     }
-
-    func onPlayerStatus(position: Int, playing: Bool) {
-        let playerId = players[position].playerId
-        players[position].isPlaying = playing
-        presenter?.markPlayerAsPlaying(withObjectId: playerId, isPlaying: playing)
+    
+    fileprivate func showPlayerDeleteAlert(indexPath: IndexPath) {
+        let deleteAlert = UIAlertController(title: "Delete Player",
```

Comment:
Well, thats strange. You probably picked some old commit, because this string is localized already.
```swift
fileprivate func showPlayerDeleteAlert(indexPath: IndexPath) {
        let deleteAlert = UIAlertController(title: "dialog.player_delete.title".localized,
                                            message: String(format: "dialog.player_delete.message".localized,
                                                            players[indexPath.row].playerName ?? ""),
preferredStyle: .actionSheet)
```